### PR TITLE
start workflow for period and some updates based on feedback

### PR DESCRIPTION
**Pull request by keith-ferney**

### CODE REVIEW

File: app/controllers/documents_controller.rb [hunk 1]
```diff
@@ -220,10 +220,12 @@ def update
       end
       if params[:publish] == "true" && @organization.enable_workflows && user
         if @document.workflow_step_id && @document.assigned_to?(user)
-          WorkflowMailer.step_email(user, @organization, @document.workflow_step.slug, component_allowed_liquid_variables(@document.workflow_step, user,@organization)).deliver_later
+          WorkflowMailer.step_email(@document, request.env["SERVER_NAME"],user, @organization, @document.workflow_step.slug, component_allowed_liquid_variables(@document.workflow_step, user,@organization)).deliver_later
```
Comment (llamapope) [inline app/controllers/documents_controller.rb]: This should be the org path (from slugs)

File: app/controllers/periods_controller.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
+class PeriodsController < OrganizationsController
+  skip_before_action :require_admin_permissions
+  skip_before_action :require_organization_admin_permissions
```
Comment (llamapope) [inline app/controllers/periods_controller.rb]: should this be here?

File: app/views/admin/periods/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  <dt><strong><%= model_class.human_attribute_name(:slug) %>:</strong></dt>
+  <dd><%= @period.slug %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:organization_id) %>:</strong></dt>
+  <dd><%= @period.organization.slug if @period.organization_id %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:start_date) %>:</strong></dt>
+  <dd><%= @period.start_date %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:duration) %>:</strong></dt>
+  <dd><%= @period.duration.to_s + " days" if @period.duration %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:is_default) %>:</strong></dt>
+  <dd><%= @period.is_default %></dd
```
Comment (llamapope) [inline app/views/admin/periods/show.html.erb]: syntax?

File: app/views/periods/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  <dt><strong><%= model_class.human_attribute_name(:slug) %>:</strong></dt>
+  <dd><%= @period.slug %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:organization_id) %>:</strong></dt>
+  <dd><%= @period.organization.slug if @period.organization_id %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:start_date) %>:</strong></dt>
+  <dd><%= @period.start_date %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:duration) %>:</strong></dt>
+  <dd><%= @period.duration.to_s + " days" if @period.duration %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:is_default) %>:</strong></dt>
+  <dd><%= @period.is_default %></dd
```
Comment (llamapope) [inline app/views/periods/show.html.erb]: syntax

File: app/controllers/periods_controller.rb (hunk 2 above)
Comment (keith-ferney) [reply app/controllers/periods_controller.rb]: yes it should

File: app/controllers/documents_controller.rb (hunk 1 above)
Comment (keith-ferney) [reply app/controllers/documents_controller.rb]: ok

**Pull request closed without merging**